app/pages/feedbacks.php

```diff
@@ -0,0 +1,27 @@
+<?php
+restrict_access();
+$user = User::getMain();
+$v = new Validator();
+$description = $v->textarea("description")->placeholder("Description")->required();
+
+if ($v->valid()) {
+    $new_feedback = new Feedback();
+    $new_feedback->user = $user;
+    $new_feedback->description = $description->value;
+    em()->persist($new_feedback);
+    em()->flush();
+    Toast::success("Merci pour votre retour !");
+    redirect("/feedbacks");
```

Comment:
Another way to achieve this:
```suggestion
    Toast::success("Merci pour votre retour !");
    $v->success("Message  envoyé");
    $description->value = null;
```
The way you used works, just wanted to showcase something else